extra/release_tool.py

```diff
@@ -112,8 +120,14 @@ def set_custom_component_maps(self, maps):
     @staticmethod
     def _initialize_component_maps():
         if Component.COMPONENT_MAPS is None:
-            with open(os.path.join(integration_dir(), "component-maps.yml")) as fd:
-                Component.COMPONENT_MAPS = yaml.safe_load(fd)
+            component_maps = execute_git(
+                None,
+                integration_dir(),
+                ["show", f"{Component.integration_version()}:component-maps.yml"],
+                capture=True,
+                capture_stderr=True,
+            )
+            Component.COMPONENT_MAPS = yaml.safe_load(component_maps)
```

Comment:
`in_integration_version` is not guaranteed to be set. You need to keep the original code path if `_integration_version is None`.

Reply:
True, fixed!